sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -26,6 +28,9 @@ const testDataEs = [
   "La carretera estaba atascada. Había mucho tráfico el día de ayer."
 ];
 
+/*
+ * TODO: Patch this back to actually use AAD before GA release
```

Comment:
can't you call skip() and disable these tests instead?

Reply:
I could, but AAD had the most extensive tests. I was basically testing everything through AAD, and just doing a few things with API Key to validate that authentication scheme. So while we're working through the AAD deployment I decided to make the AAD tests use API Key instead so that I could see if the tests were at least working for functionality.